examples/diamond_exit_code/gantry.env.yml

```diff
@@ -0,0 +1,8 @@
+version: "0"
+
+steps:
+  b:
+    exit_code: 2
+  c:
+    exit_code: 3
```

Comment:
It it is indeed as suggested and the `exit_code` in the env only overrides you could add a comment here that d's exit code is taken from the command itself

Reply:
Added a comment.